index.html

```diff
@@ -8,7 +8,6 @@
     <script src="./script.js" defer></script>
   </head>
   <body>
-
     <div class="game-board">
```

Comment:
I see that you added an image for the title of the game, but I don't see it being used. You can use `<h1><img src="" alt="Memory Card Game"></h1>` to add a heading for the page.

Reply:
Thanks for the heads up. I have included the image in my HTML file.